Replace iframe of running sessions

rgaiacs opened the pull request:
This requires a bit of work to change the background color from Grafana.

![image](https://github.com/user-attachments/assets/82dbaae6-ed07-44c2-8386-d13cca0eb5f0)

consideRatio commented on doc/about/status.rst:
```diff
@@ -43,7 +43,7 @@ The total number of user sessions running on Binder.
 .. raw:: html
 
-   <iframe src="https://grafana.mybinder.org/d-solo/fLoQvRHmk/status?panelId=6&orgId=1&theme=light" width="500" height="240" frameborder="0"></iframe>
+   <iframe src="http://grafana.mybinder.org/d-solo/3SpLQinmk/1-overview?orgId=1&from=1744897214952&to=1744908014952&timezone=browser&var-cluster=000000001&panelId=31&__feature.dashboardSceneSolo" width="500" height="240" frameborder="0"></iframe>
```
From and to, should they be hardcoded like this? I would expect from the heading that this is showing recent info, for example "last week" or "last 24 h" etc

choldgraf commented:
+1 from me once Erik's points are addressed!

rgaiacs commented:
I addressed the comments

- [x] use of HTTPS
- [x] selection of time range
- [x] background color

![image](https://github.com/user-attachments/assets/dc12bc92-c34c-4bda-8b3d-c2e523207c67)

![image](https://github.com/user-attachments/assets/6f34f067-ec5a-4318-a3c3-f147f1036234)

consideRatio reviewed (commented):
http still replaces https which works fine, but otherwise lgtm!

rgaiacs commented:
`http` replaced with `https`.

choldgraf commented:
Many thanks for the improvement to the docs @rgaiacs !
